Source/options.cpp

```diff
@@ -29,6 +29,7 @@
 #include "utils/language.h"
 #include "utils/log.hpp"
 #include "utils/paths.h"
+#include "utils/sdl_compat.h"

```

Comment:
This should no longer be necessary without the references to `SDLC_KEYSTATE_LEFTSHIFT` and `SDLC_KEYSTATE_RIGHTSHIFT`.
```suggestion
```